commands/command_call.gd

```diff
@@ -8,5 +8,9 @@ func _get_command_name() -> String:
 	return "Call"
 
 
+func _get_command_desc() -> String:
```

Comment:
Description here is only needed if there's something to show, not as placeholder